src/views/reset-password/request-password-reset.tsx

```diff
@@ -0,0 +1,65 @@
+import { Button, Card, Help, InputField, LogoIcon } from '@dhis2/ui'
+import { useAlert } from '@dhis2/app-service-alerts';
+import { useCallback, useState } from 'react'
+import { useNavigate } from 'react-router-dom'
+import { useAuthAxios } from '../../hooks'
+import styles from './reset-password.module.css'
+import { RequestPasswordResetRequest } from "../../types";
+
+export const RequestPasswordReset = () => {
+    const navigate = useNavigate()
+    const [email, setEmail] = useState('')
+    const [errorMessage, setErrorMessage] = useState('')
+    const { show: showAlert } = useAlert(
+        ({ message }) => message,
+        ({ isCritical }) => (isCritical ? { critical: true } : { success: true })
+    )
+    const onInputChange = useCallback(({value}) => {
+        // Always clear the error when user starts typing again
+        setErrorMessage('')
+        setEmail(value)
+    }, [])
+    const [{ loading }, postRequestReset] = useAuthAxios<RequestPasswordResetRequest>(
```

Comment:
Something's wrong here. You're passing a request type while useAuthAxios' type argument is the response. But since the response body is empty you can probably just leave it out.